client/src/components/DataFiles/tests/DataFiles.test.jsx

```diff
@@ -19,6 +19,10 @@ describe('DataFiles', () => {
           compress: '',
```

Comment:
This config should be below and then you need to mock fetch to provide the extract and compress app mocks

```
config: {
          extractApp: { id: 'extract', version: '0.0.1' },
          compress: { id: 'compress', version: '0.0.3' },
        },
```